### DISCUSSION TITLE
VAR1#STATE fails in 8.5.3 if the VAR1 starts with an alpha char.

### DISCUSSION THREAD

**Discussion by PabluskiNC** [arendst/Tasmota#11948] [Q&A] [answered]
I upgraded to Tasmota 8.5.3 on an Sonoff ifan3.
Now I've noticed that my rule that relies on VAR1 being set does not fire if it is set with a leading alpha character.

My rule:
on power1#state=1 do ruleTimer1 600 endon  on Rules#Timer=1 do power off endon  on power1#state=0 do ruleTimer1 0 endon  on VAR1#state do power1 1 endon  on VAR1#state do ruletimer1 600 endon

If  VAR1 is set to "A1" then the rule does not fire.
However, when set to "1A1" the rule does fire.

This seems like a bug in 8.5.3.
I tried this in 6.6.0.6 and it works as expected - fires regardless.

Anyone else hit this?

Thanks,
PabluskiNC

**Comment by arendst** [comment, marked as answer]
I suggest you update first to the latest release (9.4.0) and if it still fails report again.

**Reply by sfromis** [reply]
Just tested this on current Tasmota, 9.4, and using the command `var1 1A1` makes a rule like `on var1#state do ... endon` fire as expected.

**Comment by PabluskiNC** [comment]
I have updated to 9.4.0 and the issue is not there. Thank you.
